tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -2932,6 +2932,18 @@ inline void Ceil(const RuntimeShape& input_shape, const float* input_data,
   }
 }
 
+inline void Round(const RuntimeShape& input_shape, const float* input_data,
+                  const RuntimeShape& output_shape, float* output_data) {
+  const int flat_size = MatchingFlatSize(input_shape, output_shape);
+
+  for (int i = 0; i < flat_size; i++) {
+    int offset = i;
+    // Note that this implementation matches that of tensorFlow tf.round
+    // and corresponds to the bankers rounding method.
+    output_data[offset] = std::lrint(input_data[offset]);
```

Comment:
Don't you need FE_TONEAREST?

Reply:
@jdduke I didnt expliitly set the mode assuming the mode will be FE_TONEAREST. I understand its better to set the mode and use. Now i updated the code to set the mode. Please help to review.

Reply:
Shouldn't this be std::rint? I don't think we want a long.